src/visualize/image_like.jl

```diff
@@ -127,35 +109,114 @@ function play{T}(buffer::Array{T, 2}, video_stream, t)
     return reinterpret(T, buffer, (w,h))
 end
 
-"""
-Takes a 3D image and decides if it is a volume or an animated Image.
-"""
-function _default{T<:Colorant, X}(img::Images.Image{T, 3, X}, s::Style, data::Dict)
-    props = img.properties
-    if haskey(props, "timedim")
-        if haskey(props, "spatialorder")
-            get!(data, :spatialorder, join(props["spatialorder"], ""))
+# If the user is using the new Images, ImageAxes will be loaded
+if isdefined(Images, :ImageAxes)
+    unwrap(img::Images.ImageMeta) = unwrap(data(img))
```

Comment:
Yes, any `AbstractArray` can be converted to an `Array`. The one thing that's a bit ambiguous is whether `convert` (which is a generic function) should copy or share data: over time, julia has tried to become more consistent about this kind of thing. (For example, I did a lot of work during the 0.5 cycle to make `reshape` always return a view.) It doesn't seem that this is specified, so I've filed an issue, see https://github.com/JuliaLang/julia/issues/20826.

If it's decided that `convert` needs to have predictable copy/view behavior, then I imagine you might be interested in a package-specific zero-copy alternative? (Whenever it's possible to do, of course.)